### PR TITLE
Improved helptext of how to send mail via the systems mail subsystem

### LINKED ISSUE #2793: Send mail not only via SMTP
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

It would be great if UK would not only send mail via smtp but optional via the mail subsystem on the system it is installed at.

### ✔️ Solution

Make use of system MTA to send mail to receiver

### ❓ Alternatives

_No response_

### 📝 Additional Context

_No response_

### PR DESCRIPTION
**Pull request by apio-sys**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]:
- [X] I have read and understand the pull request rules.

# Description
Adds a help field on the notification setup screen for SMTP to make it clearer for those using local MTA.

Fixes #2793

## Type of change

Please delete any options that are not relevant.

- User interface (UI)
- Other

## Checklist

- [X] My code follows the style guidelines of this project
- [ ] I ran ESLint and other linters for modified files
- [X] I have performed a self-review of my own code and tested it
- [ ] I have commented my code, particularly in hard-to-understand areas (including JSDoc for methods)
- [X] My changes generates no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)
Screenshot from 2024-02-11 20-30-06

Please do not use any external image service. Instead, just paste in or drag and drop the image here, and it will be uploaded automatically.

### CODE REVIEW

File: src/components/notifications/SMTP.vue [hunk 1]
```diff
@@ -5,6 +5,8 @@
             <input id="hostname" v-model="$parent.notification.smtpHost" type="text" class="form-control" required>
         </div>
 
+        <div class="form-text">{{ $t("HostnameDesc") }}</div>
+
```
Comment (CommanderStorm) [inline src/components/notifications/SMTP.vue:9]:
```suggestion
        <i18n-t tag="div" keypath="Either enter the hostname of the server you want to connect to or localhost if you intend to use a locally configured mail transfer agent" class="form-text">
            <template #localhost>
                <code>localhost</code> 
            </template>
            <template #local_mta>
                <a href="https://wikipedia.org/wiki/Mail_Transfer_Agent" target="_blank">{{ $t("locally configured mail transfer agent") }}</a> 
            </template>
        </i18n-t>
```

File: src/lang/en.json [hunk 2]
```diff
@@ -57,6 +57,7 @@
     "Friendly Name": "Friendly Name",
     "URL": "URL",
     "Hostname": "Hostname",
+    "HostnameDesc": "Either enter the hostname of the SMTP server you want to use or localhost if you intend to use a locally configured MTA.",
```
Comment (CommanderStorm) [inline src/lang/en.json:60]:
```suggestion
    "locally configured mail transfer agent": "locally configured mail transfer agent",
    "Either enter the hostname of the server you want to connect to or localhost if you intend to use a locally configured mail transfer agent": "Either enter the hostname of the server you want to connect to or {localhost} if you intend to use a {local_mta}",
```

**Review by CommanderStorm — COMMENTED** [review]
What do you think about this text instead?

This changes
- `localhost` is styled like code
- adds a link to wikipedia, for those don't know what an MTA is and just want to get mails set up
- changes the translation key to be basically the same to help our translators

File: src/components/notifications/SMTP.vue (hunk 1 above)
Comment (apio-sys) [reply src/components/notifications/SMTP.vue:9]: Looking good to me. I thought I needed to create the translate key. Thanks for the corrections.

Comment (CommanderStorm) [reply src/components/notifications/SMTP.vue]: >  I thought I needed to create the translate key

You did:
In your case `HostnameDesc` was a valid translation key that you created by adding it to `en.json` ^^

File: src/components/notifications/SMTP.vue [hunk 3]
```diff
@@ -5,6 +5,14 @@
             <input id="hostname" v-model="$parent.notification.smtpHost" type="text" class="form-control" required>
         </div>
 
+        <i18n-t tag="div" keypath="Either enter the hostname of the server you want to connect to or localhost if you intend to use a locally configured mail transfer agent" class="form-text">
+            <template #localhost>
+                <code>localhost</code>
```
Comment (CommanderStorm) [inline src/components/notifications/SMTP.vue:10]:
```suggestion
                <code>localhost</code>
```

File: src/components/notifications/SMTP.vue [hunk 4]
```diff
@@ -5,6 +5,14 @@
             <input id="hostname" v-model="$parent.notification.smtpHost" type="text" class="form-control" required>
         </div>
 
+        <i18n-t tag="div" keypath="Either enter the hostname of the server you want to connect to or localhost if you intend to use a locally configured mail transfer agent" class="form-text">
+            <template #localhost>
+                <code>localhost</code> 
+            </template>
+            <template #local_mta>
+                <a href="https://wikipedia.org/wiki/Mail_Transfer_Agent" target="_blank">{{ $t("locally configured mail transfer agent") }}</a>
```
Comment (CommanderStorm) [inline src/components/notifications/SMTP.vue:13]:
```suggestion
                <a href="https://wikipedia.org/wiki/Mail_Transfer_Agent" target="_blank">{{ $t("locally configured mail transfer agent") }}</a>
```

**Comment by CommanderStorm** [comment]
Thanks for the bugfix!

All changes in this PR are small and uncontroversial ⇒ merging with junior maintainer approval

**Pull request merged**